centreon/src/Core/Notification/Infrastructure/API/AddNotification/AddNotificationPresenter.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types=1);
+
+namespace Core\Notification\Infrastructure\API\AddNotification;
+
+use Centreon\Domain\Log\LoggerTrait;
+use Core\Application\Common\UseCase\AbstractPresenter;
+use Core\Application\Common\UseCase\CreatedResponse;
+use Core\Infrastructure\Common\Presenter\PresenterFormatterInterface;
+use Core\Notification\Application\UseCase\AddNotification\AddNotificationResponse;
+
+class AddNotificationPresenter extends AbstractPresenter
+{
+    use LoggerTrait;
+
+    public function __construct(
+        PresenterFormatterInterface $presenterFormatter,
+    ) {
+        parent::__construct($presenterFormatter);
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function present(mixed $data): void
+    {
+        if (
+            $data instanceof CreatedResponse
+            && $data->getPayload() instanceof AddNotificationResponse
+        ) {
```

Comment:
Better to use a dedicated `AddNotificationPresenterInterface`.
See the `AddHostGroup` usecase for an existing example.